recipes-bsp/tbtadm/tbtadm_0.9.3.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "Thunderbolt user-space management tool"
+DESCRIPTION = "tbtadm provides convenient way to interact with Thunderbolt kernel module, approve the connection of Thunderbolt devices, handle the ACL for auto-connecting devices and more."
+HOMEPAGE = "https://github.com/intel/thunderbolt-software-user-space"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://COPYING;md5=565a8f58cc6472cf9c3ee9453d7e5aa4"
+
+DEPENDS = "cmake-native udev boost"
+RDEPENDS:${PN} = "boost-filesystem"
+
+inherit cmake
+
+SRCREV = "fe0fa2237b971ec8baae36b785d98a772684e5e7"
+SRC_URI = "git://github.com/intel/thunderbolt-software-user-space.git;protocol=https \
+           file://0001-tbtadm-Disable-manpage-target.patch \
```

Comment:
Can we get rid of this .patch and just pack the man pages into a `tbtadm-docs` subpackage?

Reply:
Not without creating a recipe for [txt2tags](https://txt2tags.org/), which is a requirement of their build process for manpages.